client/src/app/videos/+video-watch/video-share.component.ts

```diff
@@ -39,4 +42,11 @@ export class VideoShareComponent {
   notSecure () {
     return window.location.protocol === 'http:'
   }
+
+  activateCopiedMessage () {
+    this.copied = true
```

Comment:
You don't need the `copied` variable, just use the `notificationService`: https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/app/videos/%2Bvideo-watch/video-watch.component.ts#L128

You'll need to inject it in the component: https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/app/videos/%2Bvideo-watch/video-watch.component.ts#L56